packages/fxa-payments-server/src/components/PaymentForm/index.tsx

```diff
@@ -274,15 +274,17 @@ const validateName: OnValidateFunction = (value, focused) => {
 };
 
 const validateZip: OnValidateFunction = (value, focused) => {
-  let valid = true;
+  let valid = null;
```

Comment:
Looks like a type validation thing failed CI here - this should probably be `let valid = undefined;`